src/python/scripts/dx-print-bash-vars

```diff
@@ -35,4 +35,9 @@ job_input_file = file_load_utils.get_input_json_file()
 
 var_defs_hash = file_load_utils.gen_bash_vars(job_input_file)
 for key, val in var_defs_hash.iteritems():
-    print("export {}={}".format(key, val))
+    val_len = len(val)
+    if val_len <= (16 * 1024):
+        print("export {}={}".format(key, val))
+    else:
+        msg = "the variable {} is too long ({} bytes), omitting".format(key, val_len)
+        print("echo {}".format(msg))
```

Comment:
I'd also surround it in brackets otherwise it'll error out on (), this should work 
`msg = "\"the variable {} is too long ({} bytes), omitting\"".format(key, val_len)`.

Reply:
Yes, a quick test confirmation:

```
$ cat x.py 
key="a"
val_len=10
msg = "\"the variable {} is too long ({} bytes), omitting\"".format(key, val_len)
print("echo {}".format(msg))
$ python x.py > x.sh
$ source x.sh 
the variable a is too long (10 bytes), omitting
```

Reply:
Or, it'd be even better to put the quotes in the echo print